.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Denne her trenger vi vel ikke?

Reply:
Nja, kanskje ikke, men om vi vil ha det i som en ny standard i alle repo for issues, så syns jeg det er like greit å bare legge det inn med engang fra start ved nylig laget repos.